### ISSUE TITLE
Accessibility: <ul> must only directly contain <li>, <script> or <template> elements

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#225]
Another accessibility error: I ran [this page ](https://www.jccmi.edu/information-technology/e-mail/) and it's coming up with an error for the left side nav - I'm assuming this is going to show up on all pages as well that have left navs enabled. I've noticed that I have to run a lot of page checkers to find different errors, and this error showed up in some checkers but not others?

**Issue description:** Ensures that lists are structured correctly

**Element location**
#site-sidebar-left > ul.widgets

**Element source**
<ul class="widgets">

**To solve this violation, you need to fix the following:**
List element has direct children that are not allowed inside <li> elements

**Related node:**
`<div class="menu-information-technology-container">......</div>`

ul > div.menu-information-technology-container
